selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/OnpremSeleniumSuiteModule.java

```diff
@@ -104,4 +110,11 @@ public TestWorkspace getWorkspace(TestWorkspaceProvider testWorkspaceProvider,
     public ActionsFactory getActionFactory() {
         return isMac() ? new MacOSActionsFactory() : new GenericActionsFactory();
     }
+
+    @Provides
+    @Named("admin")
+    public OnpremTestOrganizationServiceClient getAdminOrganizationServiceClient(TestApiEndpointUrlProvider apiEndpointUrlProvider,
+                                                                                 TestAdminHttpJsonRequestFactory requestFactory) {
```

Comment:
We doesn't need to bind it explicitly so as we have concrete class **TestAdminHttpJsonRequestFactory** here and _Guice_ knows where it should inject it from.